cognite/client/_api/iam.py

```diff
@@ -93,6 +105,10 @@ def compare_capabilities(
     ) -> list[Capability]:
         """Helper method to compare capabilities across two groups (of capabilities) to find which are missing from the first.
 
+        Warning:
+            Capabilities that are no longer supported by the API will be ignored. These are named prefixed `Legacy` and
+            all iherit from the base class `LegacyCapability`. If you want to check for these, you must do so manually.
```

Comment:
I don't think this should be a warning
```suggestion
        Note:
            Capabilities that are no longer in use by the API will be ignored. These have names prefixed with `Legacy` and
            all inherit from the base class `LegacyCapability`. If you want to check for these, you must do so manually.
```